tutorials/12_CES_Calibration.md

```diff
@@ -0,0 +1,225 @@
+# CES Calibration
+
+## CES Production Function Basics
+
+Consider the CES production function
+
+$V_o = \left( \sum{\alpha_i {V_i}^{\rho_o}} \right)^{\frac{1}{\rho_o}}$
+
+with output quantity $V_o$, input quantities $V_i$, elasticity parameter
+$\rho_o$, and efficiency parameters $\alpha_i$.  In REMIND, the quantities are
+determined by the input data into the calibration, the elasticity parameter is
+part of the model code, and the efficiency parameters are to be determined
+during calibration.
+
+As it is not possible to calculate $n$ parameters $\alpha_i$ from a single
+equation, we use and iterative approach.  The
+[Euler identity](https://en.wikipedia.org/wiki/Homogeneous_function#Euler's_theorem)
+asserts that for homogeneous functions of degree one the function value is equal
+to the sum of the partial derivatives times the function arguments:
+
+$V_o = \sum{\frac{\partial V_o}{\partial  V_i} V_i}$
+
+As we are not seeking a general analytic solution, but only need to calculate
+values at specific points for the iterative process, we can express the partial
+derivatives in terms of the input and output quantities: 
+
+$\frac{\partial V_o}{\partial  V_i} = \pi_i = \alpha_i {V_i}^{\rho_o - 1} {V_o}^{1 - \rho_o}$
+
+Since the final output of the production function $V_o$ is also the numéraire
+(e.g. the unit – U.S. dollars – in which inputs are measured), the partial
+derivatives are the equilibrium prices $\pi_i$ of the input factors $V_i$ and
+can express the efficiency parameters in terms of quantities, prices, and
+elasticity parameters: 
+
+$\frac{\partial V_o}{\partial  V_i} = \pi_i \Leftrightarrow \alpha_i = \pi_i \left(\frac{V_o}{V_i}\right)^{1 - \rho_o}$
+
+The basic process of the calibration is to use the price calculated using the
+partial derivatives of iteration $j$ and combine them with the exogenously
+prescribed target quantities $V^\ast$ to calculate the efficiency parameters of
+the next iteration $j+1$:
+
+$\pi_i^{(j)} = \alpha_i^{(j)} {V_i^{(j)}}^{1 - \rho_o} {V_o^{(j)}}^{1 - \rho_o}$
+
+$\alpha_i^{(j+1)} = \pi_i^{(j)} \left(\frac{V_o^\ast}{V_i^\ast}\right)^{1 - \rho_o}$
+
+
+## Requirements
+
+For the calibration process to work, we need 
+
+1. trajectories for all primary production factors (`ppf`, final energy and
+   energy service demands, labour, capital) and the output (`inco`, GDP), and
+2. the previous iterations' `ppf` prices.
+
+Trajectories under (1) come from the input files `./core/input/f_gdp.cs3r`,
+`./core/input/f_pop.cs3r`,
+`./modules/29_CES_parameters/calibrate/input/f29_capitalQuantity.cs4r`, and
+`./core/input/f_fedemand.cs4r` which are generated automatically by `mrremind`
+and always present.
+
+Prices under (2) are calculated using the `input.gdx` provided to the
```

Comment:
should we mention here that usually the 'input.gdx' file is a copy of a gdx file from former runs which comes with the CES-parameters and has the name "cm_CES_configuration"

Reply:
People do know that the `input.gdx` comes from a previous REMIND run when they made it this far in the tutorials, I would argue.  I added a note on `cm_CES_configuration` below in the Results section.